.github/workflows/fmax-trials.yml

```diff
@@ -19,12 +19,12 @@ jobs:
       - run: which pip3 && which python3 && which pip
       - run: riscv64-unknown-elf-gcc --version
       - run: bash scripts/install_oxide 
```

Comment:
I wonder whether we could use a prebuilt oxide here? Maybe a nightly?

Reply:
I already have a "make-env" branch using Conda packages; it's waiting for a suitable RISC-V toolchain package.   But I suppose we can mix-and-match, using a SiFive toolchain tarball and everything else from Conda packages.

Reply:
That would definitely speed up the CI.